### ISSUE TITLE
[linter issues] repeated string in code detected by gometalinter/goconst

### ISSUE THREAD

**Issue by allencloud** [alibaba/pouch#2424]
### Ⅰ. Issue Description

[goconst](https://github.com/jgautheron/goconst) finds repeated strings that could be replaced by a constant. 

Its motivation is that there are obvious benefits to using constants instead of repeating strings, mostly to **ease maintenance**. Cannot argue against changing a single constant versus many strings.

While this could be considered a beginner mistake, across time, multiple packages and large codebases, some repetition could have slipped in.

There are the result checked by goconst:

```
apis/opts/config/blkio.go:61:9:warning: 4 other occurrence(s) of "value" found in: apis/opts/config/blkio.go:119:9 apis/opts/config/blkio.go:177:9 apis/opts/config/runtime.go:58:9 apis/opts/config/ulimit.go:43:9 (goconst)
apis/opts/config/blkio.go:119:9:warning: 4 other occurrence(s) of "value" found in: apis/opts/config/blkio.go:61:9 apis/opts/config/blkio.go:177:9 apis/opts/config/runtime.go:58:9 apis/opts/config/ulimit.go:43:9 (goconst)
apis/opts/config/blkio.go:177:9:warning: 4 other occurrence(s) of "value" found in: apis/opts/config/blkio.go:61:9 apis/opts/config/blkio.go:119:9 apis/opts/config/runtime.go:58:9 apis/opts/config/ulimit.go:43:9 (goconst)
apis/opts/config/runtime.go:58:9:warning: 4 other occurrence(s) of "value" found in: apis/opts/config/blkio.go:61:9 apis/opts/config/blkio.go:119:9 apis/opts/config/blkio.go:177:9 apis/opts/config/ulimit.go:43:9 (goconst)
apis/opts/config/ulimit.go:43:9:warning: 4 other occurrence(s) of "value" found in: apis/opts/config/blkio.go:61:9 apis/opts/config/blkio.go:119:9 apis/opts/config/blkio.go:177:9 apis/opts/config/runtime.go:58:9 (goconst)
cri/v1alpha2/cri.go:508:11:warning: 3 other occurrence(s) of "remove" found in: cri/v1alpha2/cri.go:847:11 cri/v1alpha2/cri.go:1415:11 cri/v1alpha2/cri.go:1472:11 (goconst)
cri/v1alpha2/cri.go:561:11:warning: 3 other occurrence(s) of "status" found in: cri/v1alpha2/cri.go:904:11 cri/v1alpha2/cri.go:1254:11 cri/v1alpha2/cri.go:1346:11 (goconst)
cri/v1alpha2/cri.go:637:11:warning: 2 other occurrence(s) of "list" found in: cri/v1alpha2/cri.go:867:11 cri/v1alpha2/cri.go:1302:11 (goconst)
cri/v1alpha2/cri.go:847:11:warning: 3 other occurrence(s) of "remove" found in: cri/v1alpha2/cri.go:508:11 cri/v1alpha2/cri.go:1415:11 cri/v1alpha2/cri.go:1472:11 (goconst)
cri/v1alpha2/cri.go:867:11:warning: 2 other occurrence(s) of "list" found in: cri/v1alpha2/cri.go:637:11 cri/v1alpha2/cri.go:1302:11 (goconst)
cri/v1alpha2/cri.go:904:11:warning: 3 other occurrence(s) of "status" found in: cri/v1alpha2/cri.go:561:11 cri/v1alpha2/cri.go:1254:11 cri/v1alpha2/cri.go:1346:11 (goconst)
cri/v1alpha2/cri.go:1254:11:warning: 3 other occurrence(s) of "status" found in: cri/v1alpha2/cri.go:561:11 cri/v1alpha2/cri.go:904:11 cri/v1alpha2/cri.go:1346:11 (goconst)
cri/v1alpha2/cri.go:1302:11:warning: 2 other occurrence(s) of "list" found in: cri/v1alpha2/cri.go:637:11 cri/v1alpha2/cri.go:867:11 (goconst)
cri/v1alpha2/cri.go:1346:11:warning: 3 other occurrence(s) of "status" found in: cri/v1alpha2/cri.go:561:11 cri/v1alpha2/cri.go:904:11 cri/v1alpha2/cri.go:1254:11 (goconst)
cri/v1alpha2/cri.go:1415:11:warning: 3 other occurrence(s) of "remove" found in: cri/v1alpha2/cri.go:508:11 cri/v1alpha2/cri.go:847:11 cri/v1alpha2/cri.go:1472:11 (goconst)
cri/v1alpha2/cri.go:1472:11:warning: 3 other occurrence(s) of "remove" found in: cri/v1alpha2/cri.go:508:11 cri/v1alpha2/cri.go:847:11 cri/v1alpha2/cri.go:1415:11 (goconst)
daemon/mgr/container_monitor.go:29:10:warning: 2 other occurrence(s) of "none" found in: daemon/mgr/network_utils.go:25:17 daemon/mgr/spec_linux.go:383:17 (goconst)
daemon/mgr/network_utils.go:25:17:warning: 2 other occurrence(s) of "none" found in: daemon/mgr/spec_linux.go:383:17 daemon/mgr/container_monitor.go:29:10 (goconst)
daemon/mgr/spec_linux.go:383:17:warning: 2 other occurrence(s) of "none" found in: daemon/mgr/network_utils.go:25:17 daemon/mgr/container_monitor.go:29:10 (goconst)
daemon/mgr/spec_mount.go:76:8:warning: 3 other occurrence(s) of "shared" found in: daemon/mgr/spec_mount.go:77:19 daemon/mgr/spec_mount.go:78:33 daemon/mgr/spec_mount.go:81:19 (goconst)
daemon/mgr/spec_mount.go:76:18:warning: 2 other occurrence(s) of "rshared" found in: daemon/mgr/spec_mount.go:77:43 daemon/mgr/spec_mount.go:81:43 (goconst)
daemon/mgr/spec_mount.go:77:43:warning: 2 other occurrence(s) of "rshared" found in: daemon/mgr/spec_mount.go:76:18 daemon/mgr/spec_mount.go:81:43 (goconst)
daemon/mgr/spec_mount.go:77:19:warning: 3 other occurrence(s) of "shared" found in: daemon/mgr/spec_mount.go:76:8 daemon/mgr/spec_mount.go:78:33 daemon/mgr/spec_mount.go:81:19 (goconst)
daemon/mgr/spec_mount.go:78:33:warning: 3 other occurrence(s) of "shared" found in: daemon/mgr/spec_mount.go:76:8 daemon/mgr/spec_mount.go:77:19 daemon/mgr/spec_mount.go:81:19 (goconst)
daemon/mgr/spec_mount.go:80:17:warning: 2 other occurrence(s) of "rslave" found in: daemon/mgr/spec_mount.go:81:91 daemon/mgr/spec_mount.go:82:33 (goconst)
daemon/mgr/spec_mount.go:81:19:warning: 3 other occurrence(s) of "shared" found in: daemon/mgr/spec_mount.go:76:8 daemon/mgr/spec_mount.go:77:19 daemon/mgr/spec_mount.go:78:33 (goconst)
daemon/mgr/spec_mount.go:81:43:warning: 2 other occurrence(s) of "rshared" found in: daemon/mgr/spec_mount.go:76:18 daemon/mgr/spec_mount.go:77:43 (goconst)
daemon/mgr/spec_mount.go:81:91:warning: 2 other occurrence(s) of "rslave" found in: daemon/mgr/spec_mount.go:80:17 daemon/mgr/spec_mount.go:82:33 (goconst)
daemon/mgr/spec_mount.go:82:33:warning: 2 other occurrence(s) of "rslave" found in: daemon/mgr/spec_mount.go:80:17 daemon/mgr/spec_mount.go:81:91 (goconst)
daemon/mgr/system.go:64:19:warning: 3 other occurrence(s) of "<unknown>" found in: daemon/mgr/system.go:90:14 daemon/mgr/system.go:104:12 daemon/mgr/system.go:184:19 (goconst)
daemon/mgr/system.go:90:14:warning: 3 other occurrence(s) of "<unknown>" found in: daemon/mgr/system.go:64:19 daemon/mgr/system.go:104:12 daemon/mgr/system.go:184:19 (goconst)
daemon/mgr/system.go:104:12:warning: 3 other occurrence(s) of "<unknown>" found in: daemon/mgr/system.go:64:19 daemon/mgr/system.go:90:14 daemon/mgr/system.go:184:19 (goconst)
daemon/mgr/system.go:184:19:warning: 3 other occurrence(s) of "<unknown>" found in: daemon/mgr/system.go:64:19 daemon/mgr/system.go:90:14 daemon/mgr/system.go:104:12 (goconst)
```

### Ⅱ. Describe what happened

goconst detected lots of repeated string rather than an instance. 

### Ⅲ. Describe what you expected to happen
no repeated strings.

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by mathspanda** [comment]
During working on this, I think the `unknown` values shouldn't be returned in /info response. It is just a presentation, and should be rendered in pouch cli.

**Comment by fuweid** [comment]
has been fixed by #2454
